posixlib/src/main/resources/scala-native/sys/socket.c

```diff
@@ -76,6 +76,8 @@ int scalanative_so_rcvtimeo() { return SO_RCVTIMEO; }
 
 int scalanative_so_reuseaddr() { return SO_REUSEADDR; }
 
+int scalanative_so_reuseport() { return SO_REUSEPORT; }
```

Comment:
This seems to be failing on windows. I didn't realize posixlib was compiled for windows.

In any case, what is the correct way to not compile this on Windows?

Reply:
Since this is C code, I believe (know) that you can use `#ifdef`.
Just do not admit to anyone that you are using a preprocessor, especially the C preprocessor.

`./posixlib/src/main/resources/scala-native/sys/socket.c`
shows two ways to handle the situation.  

One is the `#ifdef _WIN32` at the top of the file, which would probably
want to be `#ifdef/#else `.

The second is testing for the macro name itself, see MSG_NOSIGNAL
in that file.

I can take a few minutes to look around. I think I have encountered/written a better example.  Just have to find it.
Have only a few minutes while CI gets through 'lint'

@ me if neither of these work. I think the latter method is
likely to be more robust to the next random OS that is added to SN.
Just me being a curmudgeon at this late hour.

Reply:
PS: please do something good, so that I can crib it for 0.5.0.
      What is the expression about imitation being the best form
      of regard?

Reply:
Thanks! Seems to me that testing for the macro itself is most robust, so I did that.

Reply:
Good to hear that you got unstuck.   Helps me feel like I did something today.

For future reference & probably more than you ever wanted to know, _WIN32 covers
both 32 & 64 bit Windows, despite the 32 in the name.